server/src/main/java/org/elasticsearch/common/unit/RelativeByteSizeValue.java

```diff
@@ -9,14 +9,19 @@
 package org.elasticsearch.common.unit;
 
 import org.elasticsearch.ElasticsearchParseException;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+
+import java.io.IOException;
 
 /**
  * A byte size value that allows specification using either of:
  * 1. Absolute value (200GB for instance)
  * 2. Relative percentage value (95%)
  * 3. Relative ratio value (0.95)
  */
-public class RelativeByteSizeValue {
+public class RelativeByteSizeValue implements Writeable {
```

Comment:
Could we add some tests to `org.elasticsearch.common.unit.RelativeByteSizeValueTests` to show that this round-trips over the wire correctly?